src/test/java/core/basesyntax/service/ParseServiceImplTest.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import org.junit.Test;
+
+public class ParseServiceImplTest {
+    private static final String VALID_TEST_DATA = "s,apple,100";
+    private static final String NOT_VALID_TEST_DATA = "s,apple,-100";
+
+    @Test
+    public void parseValidData_Ok() {
```

Comment:
```suggestion
    public void parseValidData_Ok() {
```
method names should be like this `<name_of_tested_method>_<special_state_that_you_check>_<Ok/NotOk>`
fix naming in every test class